### DISCUSSION TITLE
PSReadline error

### DISCUSSION THREAD

**Discussion by gihanrangana** [lzybkr/PSReadLine#2995] [Q&A] [answered]
why this??
![image](https://user-images.githubusercontent.com/24897554/141062116-22f7ff4f-5722-4438-9a54-6c20bdcefe39.png)

**Comment by iSazonov** [comment]
Please update latest Beta4 version.

**Comment by daxian-dbw** [comment, marked as answer]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.

**Reply by gihanrangana** [reply]
thanks now it's working
